scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -49,12 +49,11 @@ fi
 WORKSPACE=/root/project
 
 # Increase nodejs stack size
-if [ -e ~/.emscripten ]
-then
-    sed -i -e 's/NODE_JS="nodejs"/NODE_JS=["nodejs", "--stack_size=8192"]/' ~/.emscripten
-else
-    echo 'NODE_JS=["nodejs", "--stack_size=8192"]' > ~/.emscripten
-fi
+test -e /emsdk_portable/node/bin/node_orig || (
+  mv /emsdk_portable/node/bin/node /emsdk_portable/node/bin/node_orig
+  echo -e '#!/bin/sh\nexec /emsdk_portable/node/bin/node_orig --stack-size=8192 $@' > /emsdk_portable/node/bin/node
+  chmod 755 /emsdk_portable/node/bin/node
+)
```

Comment:
in Bash, the `( ... )` actually spawns a new process, a sub-shell. That's not really required here. I'd suggest to also stick with the more intuitive version: `if`-statement. Also, actions on to be invoked when a negated test I personally find rather  less intuitive to read. (IMHO).